packages/examples/old/various/logo.js

```diff
@@ -5,20 +5,27 @@
 // tags       : Logo,Intersection,Sphere,Cube,Big
```

Comment:
Why this change?

Reply:
Hah, used it to generate the background.  I'll revert